fuse/mount_linux.go

```diff
@@ -189,7 +189,11 @@ func unmount(mountPoint string, opts *MountOptions) (err error) {
 		return err
 	}
 	errBuf := bytes.Buffer{}
-	cmd := exec.Command(bin, "-u", mountPoint)
+	args := []string{"-u", mountPoint}
+	if opts.LazyUnmount {
+		args = append(args, "-z")
+	}
```

Comment:
```
	args := []string{"-u", mountPoint}
	if opts.LazyUnmount {
		args[0] = "-uz"
	}
```

Looks better to me as it avoids re-sizing the size

Reply:
Indeed